README.md

```diff
@@ -28,9 +34,9 @@ The repository has no (pre)releases yet, work in progress is within the main bra
 
 ## Contributing
 * Meetings are held virtually <!-- for new API families request a meeting link from the LF admin team or replace the information with the existing meeting information (of the API family) -->
-    * Schedule: !! TBD
+    * Schedule: bi-weekly, Mondays, 10 AM CET (09:00 UTC, 08:00 UTC during European DST). For date/time of the next meeting, see previous meeting minutes
     * [Registration / Join]() !! Update this link with your meeting registration/join link
```

Comment:
Meetings now need to be fixed to UTC. So you need to choose 08:00 UTC or 09:00 UTC and stick to it.

It would be better to have the meetings set-up in Zoom and include the link here before merging thus. If you e-mail support@camaraproject.org, they will set-up the Zoom meeting schedule for you.